core/common/src/main/java/alluxio/AbstractThriftClient.java

```diff
@@ -117,7 +117,7 @@
         }
         exception = new TException(ae);
       } catch (TException e) {
-        LOG.error(e.getMessage(), e);
+        LOG.warn(e.getMessage(), e);
```

Comment:
I would prefer to have some English in the message, in some cases `e.getMessage` might be nonsense.